extension/src/openvic-extension/classes/GUINode.hpp

```diff
@@ -73,5 +88,15 @@ namespace OpenVic {
 		static godot::String int_to_formatted_string(int64_t val);
 		static godot::String float_to_formatted_string(float val, int32_t decimal_places);
 		static godot::String format_province_name(godot::String const& province_identifier);
+
+		godot::Ref<godot::BitMap> get_click_mask() const;
+		void set_click_mask(godot::Ref<godot::BitMap> const& mask);
+
+		void set_click_mask_from_nodepaths(godot::TypedArray<godot::NodePath> const& paths);
+		bool _update_click_mask_for(godot::Ref<godot::Image>& img, int index);
+		void update_click_mask();
+		void _handle_click_mask_children_gui_input(const godot::Ref<godot::InputEvent>& event);
+
+		bool _has_point(const godot::Vector2& point) const override;
```

Comment:
```suggestion
		bool _update_click_mask_for(godot::Ref<godot::Image> const& img, int index);
		void update_click_mask();

		bool _has_point(godot::Vector2 const& point) const override;
```
- `_update_click_mask_for`'s `img` argument can be `const&` as the `Ref` itself isn't changed (same change needed in the cpp).
- `_handle_click_mask_children_gui_input` doesn't exist anywhere else.
- `_has_point`'s `point` argument should have the `const` after the type (same change needed in the cpp).